src/App.js

```diff
@@ -1,10 +1,38 @@
 import 'antd/dist/antd.css';
 import React, { Suspense } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 function App() {
   return (
     <div className="App">
-      <Suspense fallback="loading"></Suspense>
+      <Router>
```

Comment:
can you add the Navbar here too?

Reply:
Sure, it was there while testing it... I'll re-add it